nautobot/core/management/commands/build_ui.py

```diff
@@ -137,5 +137,6 @@ def handle(self, *args, **options):
         if options["npm_build"]:
             args = f"run build {loglevel}"
             self.run_command(f"npm {args}", ">>> Compiling Nautobot UI packages...")
+            self.run_command(f"cp {Path(settings.NAUTOBOT_UI_DIR, 'build' ,'asset-manifest.json')} {Path(settings.NAUTOBOT_UI_DIR, 'build' , 'static', 'asset-manifest.json')}", ">>> Copying built manifest...")
```

Comment:
This file seems to be required for the React Developer Tools (Chrome/Firefox plugin) to work.

Reply:
Ah, in that case instead of copying it, let's have it be a static file that is put into place by Webpack. Can we please backlog this? It can be done using [`CopyWebpackPlugin`](https://webpack.js.org/plugins/copy-webpack-plugin/).